users/edit-alias.php

```diff
@@ -22,6 +22,12 @@
 require_once('../common.php');
 $smarty->assign ('smarty_template', 'users_edit-alias');
 
+// is edit-alias support enabled in $CONF ?
+if($CONF['edit_alias'] == 'NO') {
```

Comment:
Please change this condition to
`if (! Config::bool('edit_alias')) {`

Also, please move this section after `$USERID_USERNAME = ...` so that the authentification check stays the first thing edit-alias does.